docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -123,6 +123,22 @@ an external service called the {package-registry}.
 
 For this to work, the {kib} server must connect to `https://epr.elastic.co` on port `443`.
 
+[discrete]
+[[fleet-errors-tls]]
+== {kib} cannot connect to {package-registry} in air-gapped environments
+
+In air-gapped environments where custom Certificater Authorities (CA) are used, if the custom
+CA is not available for {kib}, you can see error like this one:
+
+[source,json]
+---
+{"type":"log","@timestamp":"2022-03-02T09:58:36-05:00","tags":["error","plugins","fleet"],"pid":58716,"message":"Error connecting to package registry: request to https://customer.server.name:8443/categories?experimental=true&include_policy_templates=true&kibana.version=7.17.0 failed, reason: self signed certificate in certificate chain"}
+---
+
+This can be addressed by adding your CA certificate file path to the {kib} startup
+file by defining the `NODE_EXTRA_CA_CERTS` environment variable.
+
+
```

Comment:
How can I set a link to the other page ? So the user could see more information about this.

Reply:
Try with `<<air-gapped>>`, or `<<air-gapped-tls>>` for the section about TLS.

Reply:
Perfect, I'll add a link with that notation, thanks!